### PR TITLE
ESR Board Intake Approvals

### PR DESCRIPTION
**Pull request by sarwarahmed1**
Test cases covering ESR Board member approvals during the ESR Board Intake Review stage. Also includes some minor revisions and space fixes.

### CODE REVIEW

**Review by matakevin — APPROVED** [review]
LGTM

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java [hunk 1]
```diff
@@ -20,43 +20,43 @@ public static void clickOnIntakePreparationCatalogTask() {
...
+        Playwright_Common_Locators.iframeLocator().getByText("Hardware").click();
+        Playwright_Common_Locators.iframeLocator().getByText("Other", new FrameLocator.GetByTextOptions().setExact(true)).click();
+        Playwright_Common_Locators.iframeLocator().getByRole(AriaRole.TEXTBOX, new FrameLocator.GetByRoleOptions().setName("   Other Funding Type")).click();
+        Playwright_Common_Locators.iframeLocator().getByRole(AriaRole.TEXTBOX, new FrameLocator.GetByRoleOptions().setName("   Other Funding Type")).fill("Test");
+        Playwright_Common_Locators.iframeLocator().getByText("Yes").nth(1).click();
+        Playwright_Common_Locators.iframeLocator().getByText("Yes").nth(2).click();
+        Playwright_Common_Locators.iframeLocator().getByRole(AriaRole.LINK, new FrameLocator.GetByRoleOptions().setName(" Choose date...")).nth(1).click();
+        Playwright_Common_Locators.iframeLocator().getByLabel("Go to Today", new FrameLocator.GetByLabelOptions().setExact(true)).click();
+        Playwright_Common_Locators.iframeLocator().getByRole(AriaRole.LINK, new FrameLocator.GetByRoleOptions().setName(" Choose date...")).nth(2).click();
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java:50]: Is the question mark that looks like an image, interactable by playwright? 
image

**Review by hamidnesar — APPROVED** [review]
Approved!

**Review by katm007 — APPROVED** [review]
Approved!

File: src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java (hunk 1 above)
Comment (sarwarahmed1) [reply src/test/java/ServiceNow/ESR/Playwright/StepsImplementation/ESRTaskStepsImplementation.java:50]: Yup it is!

**Pull request merged**